### ISSUE TITLE
Connection lost to the server once switch phone connectivity

### ISSUE THREAD

**Issue by sigma2017** [vector-im/element-android#6852]
### Steps to reproduce

Hello, once I switch on one phone from wifi to 5g or vice versa, the receiver is not able anymore to see  text messages, normal chat
It takes a lot of time for the text to arrive like 10 minutes or something.
If a call is started, one phone will stay in "connecting" and the other in "ringing".

### Outcome

This is not a normal behavior, I can't rely on this app in these circumstances.
Note: One small update, I just pip install --upgrade matrix-synapse and the server upgraded from 1.64.0 to 1.65.0 and the bahavior looks to be the same
I am using also coturn integrated into synapse following the official server documentation.

I was too excited about this solution to have a private messenger but you can NOT rely on it.

### Your phone model

Samsung A53 5g

### Operating system version

ANdroid12

### Application version and app store

element 1.4.32 downloaded from here

### Homeserver

 matrix-synapse-1.64.0

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by ouchadam** [comment]
thanks for raising, this may be related to the background sync issues we've recently fixed https://github.com/vector-im/element-android/issues/6884 , could I ask that you try the latest release [1.4.34](https://github.com/vector-im/element-android/releases/tag/v1.4.34) and see if the issue is resolved?

**Comment by sigma2017** [comment]
Hello, to be honest I haven't after I noticed that SchildiChat hasn't this issue and after updating to [sc_v1.4.34.sc58] which is based on element is even better.
The main issue which still remains is the fact that when somebody calls you, MOST of the times nothing happens and you should open the phone screen, launch the application in order to answer, not to mention about the notifications which arrive with a delay.
I will install element back to test the problem I opened this issue in the first place and I will come back with an answer.
Thanks.

**Comment by sigma2017** [comment]
Hello, I confirm the issue was fixed.

**Comment by ouchadam** [comment]
thanks for confirming the VOIP part of the issue is fixed :+1:

**Comment by Florian14** [comment]
Hi @sigma2017, sorry for the lack of activity here, can you give an update about the issue? Do you still observe the issue?

**Comment by sigma2017** [comment]
Hello Florian14,
I am using SchildiChat since then which is based on element and there is no issue anymore.
I have some friends on my server and all of them are using element, the issue was fixed in the mean time.
Also I have updated the server, many versions changed during all this time.
I have a separated dedicated VM with coturn, running Zero SSL certificate and the call looks stable as well.

Thank you, the issue can be closed.
